layers/exporters/matrix_gen.py

```diff
@@ -0,0 +1,332 @@
+import os
+import requests
+from stix2 import TAXIICollectionSource, Filter, MemoryStore, FileSystemSource
+from taxii2client import Server, Collection
+
+class MatrixEntry:
+    def __init__(self, id=None, name=None):
+        if id is not None:
+            self.id = id
+        if name is not None:
+            self.name = name
+
+    @property
+    def id(self):
+        if self.__id is not None:
+            return self.__id
+
+    @id.setter
+    def id(self, id):
+        self.__id = id
+
+    @property
+    def name(self):
+        if self.__name is not None:
+            return self.__name
+
+    @name.setter
+    def name(self, name):
+        self.__name = name
+
+    @property
+    def score(self):
+        if self.__score is not None:
+            return self.__score
+
+    @score.setter
+    def score(self, score):
+        self.__score = score
+
+class MatrixColumn:
```

Comment:
Why isn't `MatrixColumn` called `Tactic`? Like, that's what a column in the matrix represents: a tactic, and the contents of the tactic.

Reply:
I think this is left over from when it was a 1-1 relationship between column entries and the final matrix, so subtechniques had their own column. I'll update it since that's no longer the case.